ep_testing/tests/api.py

```diff
@@ -54,44 +63,63 @@ def run(self, install_root: str, verbose: bool, kwargs: dict):
             f.write(self._api_script_content(install_root))
         print(' [FILE WRITTEN] ', end='')
         try:
-            if platform.system() == 'Linux':
-                py = 'python3'
-            elif platform.system() == 'Darwin':
-                py = '/usr/local/bin/python3'
-            else:  # windows
-                py = 'C:\\Python36\\Python.exe'
+            # TODO: why do we have to pass the full path to python on darwin/Windows but not linux?
+            # Anyhoo, let's just use sys.executable everywhere
+            # if platform.system() == 'Linux':
+            #     py = 'python3'
+            # elif platform.system() == 'Darwin':
+            #     py = '/usr/local/bin/python3'
+            # else:  # windows
+            #    py = 'C:\\Python36\\Python.exe'
+            py = sys.executable
```

Comment:
Don't harcorde the path to python. Assuming a full path is needed, just use the current full path